README.md

```diff
@@ -1,2 +1,325 @@
+Write ndjson data into the collection, each line in ndjson is regarded as a record, if here exist some indexes, the indexes will be inserted as well.
+
+### `func (collection *Collection) Iterate(ctx context.Context) (<-chan Record, error)`
+Iterate all records in the collection, it is not recommanded.
+
+### `func (db *Database) ExportToFile(ctx context.Context, destination string) error`
+Export all data of DB into the car file.
+
+### `func ImportFromFile(source string) (*Database, error)`
+Load database from the car file.
+
+### `func Merge(ctx context.Context, db *Database, other *Database) (*Database, error)`
+Merge two DBs includes ProllyTree and raw data in blockstore.
+### `func (collection *Collection) Search(ctx context.Context, query Query) (<-chan Record, error)`
+Search records using the query.Return the channel that closes while end
+
+### `func (collection *Collection) CreateIndex(ctx context.Context, fields ...string) (*Index, error)`
+Create an index for a set of fields.This will go over all the documents inserted after the index created in the collection and if they have the apropriate fields, will add them to the index.Indexing fields is important if you want to sort based on a query, or want to take advantage of sparsely querying the dataset to speed things up.
+
+### `func (collection *Collection) GetProof(recordId []byte) (*InclusionProof, error)`
+Get proof of a record in ProllyTree by its recordId, it includes recordKey, prollytree root cid and the path that recordKey in the tree.
+
+### `func (db *Database) ExportProof(ctx context.Context, prfCid cid.Cid, destination string) error`
+Export the proof of a record and related blocks
+
+### `func (collection *Collection) Get(ctx context.Context, recordId []byte) (ipld.Node, error)`
+Get data by the recordId
+
+### `func (collection *Collection) Indexes(ctx context.Context) ([]Index, error)`
+Return all indexes of the collection
+
+### `func (collection *Collection) Insert(ctx context.Context, record ipld.Node) error`
+Insert a record into DB including raw data and index creating
+
+### `func (db *Database) GetDBMetaInfo() (*schema.DBMetaInfo, error)`
+Get metainfo about the database
+
+### `func (record *Record) AsIPLDWithProof(proof tree.Proof) (ipld.Node, error)`
+Encode record and its proof as ipld node.
+
+### `func (db *Database) RootCid() cid.Cid`
+
+### `func (db *Database) GetBlockstore() *blockstore.Blockstore`
+
+### `func (index *Index) Fields() []string`
+
+### `func (index *Index) Exists() bool`
+
+### `func (collection *Collection) BestIndex(ctx context.Context, query Query) (*Index, error)`
+Choose best index for a query
+
+### `func (query Query) Matches(record Record) bool`
+Return whether the record matches the query or not
+
+### `func (record *Record) AsIPLD() (datamodel.Node, error)`
+Return record as IPLD node
+
+### `func ParseStringsFromCBOR(data []byte) ([]string, error)`
```

Comment:
These last ones should be removed from the docs since they're only relevant to advanced uses